.envs-rabbitmq

```diff
@@ -1,3 +1,4 @@
 # envs-rabbitmq
 RABBITMQ_USERNAME=georchestra
 RABBITMQ_PASSWORD=georchestra
+RABBITMQ_PORT=15672
```

Comment:
I propose to add a comment before explaining that it only change the port for the applications consuming rabbitmq and **doesn't alter the actual rabbitmq server inside docker**!

Also rabbitmq port default port is 5672 not 15672.

```suggestion
# Only apply to applications consuming rabbitmq, doesn't change the port rabbitmq server running in Docker.
RABBITMQ_PORT=5672
```